### DISCUSSION TITLE
PHP implementation

### DISCUSSION THREAD

**Discussion by eberkund** [connectrpc/.github#12] [Polls]
Would you or your company use a Connect implementation in PHP? If yes, please leave a comment and tell us more about what you'd like: synchronous or asynchronous, important framework integrations (Laravel?), and anything else important.

**Comment by omar391** [comment]
@eberkund just a bump for some updates.

**Reply by omar391** [reply]
I think if it integrates well with the "composer" ecosystem, any PHP framework can use it. Also "asynchronous" should be a priority.

**Comment by LukStankovic** [comment]
Would be super useful 🙌🏻 We want to switch from Twirp to Connect, but we can't because of the lack of support in PHP 😞

**Comment by jbendotnet** [comment]
We have numerous projects where Connect clients in PHP would be very useful, just a HTTP1 and JSON to replace OpenAPI generated clients (and avoiding the need for gRPC-GW) would be handy.
